index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:200,300,300i,400,700&display=swap" rel="stylesheet">
+
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="stylesheets/style.css">
+    <link rel="stylesheet" href="stylesheets/animate_name.css">
+
+    <title>Portfolio</title>
+</head>
+
+<body>
+
+    <!-- Header -->
+    <header class="header">
+        <div class="container">
+            <nav class="nav" id="nav">
+                <a class="nav__link" href="index.html">home</a>
+                <a class="nav__link" href="pages/portfolio.html">projects</a>
+                <a class="nav__link" href="pages/about.html">about me</a>
+                <a class="nav__link" href="pages/blog.html">blog</a>
+                <a class="nav__link" href="pages/contact.html">contact</a>
+            </nav>
+        </div>
+    </header>
+
+    <!-- Intro -->
+    <section class="intro">
+        <div class="container">
+            <div class="intro__inner">
+                <div class="intro__content">
+                    <h2 class="intro__subtitle">Hello I'm</h2>
+                    <h1 id="intro__title" data-text="Nina Patrina...">Nina Patrina</h1>
+                    <h2 class="intro__subtitle">(she/her)</h2>
+                    <div class="intro__text">Software Engineering Student <br>at Ada Developers Academy (Cohort 17)
+                    </div>
+                    <div class="social">
+                        <a class="social__link" href="https://github.com/NinaPatrina" target="_blank"
+                            rel="noopener noreferrer">
```

Comment:
Style: It's a good idea to avoid using `target="_blank"` since it takes away control from the user.  (Also, then it's not necessary to set `noopener` or `noreferrer`.)